### DISCUSSION TITLE
how to add progress bar for laravel 7 voyager "edit-add.blade.php"

### DISCUSSION THREAD

**Discussion by HadiZareZadeh** [thedevdojo/voyager#5239] [Q&A]
## Version information

 - Laravel: v7.0
 - Voyager: v1.4
 - PHP: 7.4
 - Database: MariaDB 10.4.13
 
## Description:
I'm trying to have a progress bar for my "edit-add.blade.php" from voyager breads. I've overridden blades as I want to. everything works fine, except the ajaxForm that I'm using for the progress bar. I've searched about it (maybe not enough), but couldn't find anything useful, nor in your docs. any suggestions?

**Comment by MrCrayon** [comment]
In general Voyager provides a way to manage quickly your data in an admin context, whatever you want to do custom is up to you.

You are not explaing what this progress bar is doing so it's hard to understand if there might be something Voyager related.
